jplag.frontend.scala/src/test/resources/de/jplag/scala/Complete.scala

```diff
@@ -0,0 +1,162 @@
+package mypackage
```

Comment:
I like how comprehensive this test already is, it uses a lot of custom scala features. What I'm currently missing are custom control structures. A method `method(args)` can be either called directly `method(args)` or as control structure, where the body becomes the argument `method { args }`. This could be a potential attack as every function call can be rewritten this way.

Reply:
For more information on that, this ancient blog post might be relevant: https://github.com/sebinside/sebinside-blog/blob/master/blog/md/scalafeatures-teil2.md#9-eigene-kontrollstrukturen

Reply:
That's a nice feature! I included an example. The parser luckily treats them the same.